pkg/controllers/uiplugin/plugin_info_builder.go

```
@@ -38,7 +38,8 @@ type UIPluginInfo struct {
 }
 
 var pluginTypeToConsoleName = map[uiv1alpha1.UIPluginType]string{
-	uiv1alpha1.TypeDashboards:           "console-dashboards-plugin",
+	// Deprecated: retained for console deregistration during cleanup of existing Dashboards CRs.
+	uiv1alpha1.TypeDashboards: "console-dashboards-plugin",
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gci)